autosklearn/evaluation/__init__.py

```diff
@@ -101,7 +101,8 @@ def __init__(self, backend, autosklearn_seed, resampling_strategy, metric,
                  run_obj='quality', par_factor=1, all_scoring_functions=False,
                  output_y_hat_optimization=True, include=None, exclude=None,
                  memory_limit=None, disable_file_output=False, init_params=None,
-                 budget_type=None, ta=False, pynisher_context='spawn', **resampling_strategy_args):
+                 budget_type=None, ta=False, port=9020,
```

Comment:
The port is mandatory I guess?

Reply:
Yes, I have now moved to mandatory ports